### ISSUE TITLE
Calculation Group Items Not Sorting Correctly

### ISSUE THREAD

**Issue by patrickbcox** [TabularEditor/TabularEditor3#537]
Hi,
I have a calculation group with 3 items... CY, PY, YOY.  The ordinal values are...  CY=0, PY=1, YOY=2.  Based on this, I would expect the items to sort CY, PY, YOY in my Matrix.  However when I drop the Calculation group into the Columns section of my matrix, the items are sorting... PY, CY, YOY.  I even tried creating a new calculation group and sorting is the same.  Any suggestions?  thanks.

**Comment by otykier** [comment]
Hi Patrick,
Did you perform a refresh of the calculation group table after changing the ordinal values?

**Comment by patrickbcox** [comment]
Hi, refreshing the table does not work.  And strangely, no matter the order I add my items, the same order appears in the visual.  I am on the following version of PBI... Version: 2.105.664.0 64-bit (May 2022)
Hopefully someone from TE can help as this is frustrating and I need to finish this report I am working on.
Thanks.

**Comment by otykier** [comment]
@patrickbcox I'm not sure this is a TE issue. Could you try to [capture the DAX query generated by the visual in Power BI Desktop](https://docs.microsoft.com/en-us/power-bi/create-reports/desktop-performance-analyzer#using-performance-analyzer) and share it here? Perhaps the visual is applying some unexpected sorting.

**Comment by patrickbcox** [comment]
Hi, yes, I will do that now.  But something really strange just happened.  I created calculated items all with the same expression... "SELECTEDMEASURE()" and I got the sort I wanted.  Then as soon as I updated the middle measure with the correct expression, it jumped to the front of the list.  My measures as...

CY = SELECTEDMEASURE
PY = CALCULATE(SELECTEDMEASURE(),SAMEPERIODLASTYEAR(DATES[DATE])
YOY = SELECTEDMEASURE() - CALCULATE(SELECTEDMEASURE(),SAMEPERIODLASTYEAR(Dates[Date]))

For some reason, PY is always placed in the front spot.
